+++ ./xkb/XKBGAlloc.c
@@ -239,17 +239,6 @@
 /***====================================================================***/
 
 void
-XkbFreeGeomOverlayKeys(XkbOverlayRowPtr row, int first, int count, Bool freeAll)
-{
-    _XkbFreeGeomLeafElems(freeAll, first, count,
-                          &row->num_keys, &row->sz_keys,
-                          (char **) &row->keys, sizeof(XkbOverlayKeyRec));
-    return;
-}
-
-/***====================================================================***/
-
-void
 XkbFreeGeomKeys(XkbRowPtr row, int first, int count, Bool freeAll)
 {
     _XkbFreeGeomLeafElems(freeAll, first, count,
